misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/expression/ExpressionAsProgramEvaluator.java

```diff
@@ -498,7 +497,7 @@ private List<NonTerminalNode> convertImports(List<NonTerminalNode> declarationLi
             }
 
             // Replaces original package name with the evaluation package name.
-            List<Node> moduleParts = importDeclarationNode.moduleName().stream().collect(Collectors.toList());
+            List<IdentifierToken> moduleParts = new ArrayList<>(importDeclarationNode.moduleName().stream().toList());
```

Comment:
Let's leave this as it was previously

Reply:
Fixed in [2778660](https://github.com/ballerina-platform/ballerina-lang/pull/42904/commits/277866060d74b08115089625bb7d582c095697a1)